src/libs/Network/NetworkStore.ts

```diff
@@ -82,85 +78,51 @@ Onyx.connect({
             triggerReconnectCallback();
         }
 
-        offline = Boolean(network.shouldForceOffline) || network.isOffline;
+        offline = Boolean(network.shouldForceOffline) || !!network.isOffline;
     },
 });
 
-/**
- * @returns {Object}
- */
-function getCredentials() {
+function getCredentials(): Credentials | null {
```

Comment:
See comment above, I think the intention is credentials being an object always?

Reply:
Most of the time it is an object but before the initialization credentials can be null